docs/mmteb/points/727.jsonl

```diff
@@ -0,0 +1,2 @@
+{"GitHub": "mrshu", "New dataset": 2}
```

Comment:
2 points pr. dataset and a 4 points bonus for adding a new language for bitext. I don't believe you do that so I think 2 points it is.

Reply:
Understood, thanks!